src/applications/vaos/new-appointment/components/ClinicChoicePage.jsx

```diff
@@ -45,142 +40,118 @@ const initialSchema = {
     },
   },
 };
-
 const uiSchema = {
   clinicId: {
     'ui:widget': 'radio',
   },
 };
-
 const pageKey = 'clinicChoice';
-
-export class ClinicChoicePage extends React.Component {
-  componentDidMount() {
-    this.props.openClinicPage(pageKey, uiSchema, initialSchema);
+export function ClinicChoicePage({
+  schema,
+  data,
+  facilityDetails,
+  typeOfCare,
+  clinics,
+  eligibility,
+  canMakeRequests,
+  openClinicPage,
+  updateFormData,
+  facilityDetailsStatus,
+  pageChangeInProgress,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+}) {
+  const history = useHistory();
+  const typeOfCareLabel = formatTypeOfCare(typeOfCare.name);
+  const usingUnsupportedRequestFlow =
+    data.clinicId === 'NONE' && !canMakeRequests;
+  useEffect(() => {
+    openClinicPage(pageKey, uiSchema, initialSchema);
     scrollAndFocus();
```

Comment:
I think you can remove `scrollAndFocus` here, since it'll get called in the other `useEffect` twice: once on mount, then again when your conditional in the dependency array changes.

Reply:
Good call, I wasn't sure so that answers my question too.